docs/api-docs/partner/support.md

```diff
@@ -8,23 +8,41 @@
 - [Developer library](#developer-library)
 - [Developer Slack](#developer-slack)
 - [Changelog](#changelog)
+- [BigCommerce Status](#bigcommerce-status)
 - [Town Hall](#town-hall)
+- [Partner Support Team](#partner-support-team)
 </div> 
 
+As a BigCommerce partner, you have access to a wide variety of support resources. This article outlines the different types of resources available to you.
+
 ## Developer community
+
 The [Developer community](https://support.bigcommerce.com/s/group/0F913000000HLjECAW/bigcommerce-developers) is a forum for web developers working with BigCommerce APIs, creating a custom theme, or building a custom app or integration. Find answers, browse topics, and talk to other developers in this group.
 
 ## Developer blog
+
 The [Developer blog](https://medium.com/bigcommerce-developer-blog) provides product, industry, how-to, and community information regularly. Find news, tips, and stories for developing on BigCommerce. 
 
 ## Developer library
+
 The [Developer library](https://developer.bigcommerce.com/dev-library/) provides samples, tools, and guides to help jump-start the perfect ecommerce experience. Find examples and starters to help you begin working on BigCommerce.
 
 ## Developer Slack
-BigCommerce has a dedicated Slack channel for Tech and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with members of the BigCommerce Product team. To join the BigCommerce developer Slack community, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform). 
+
+BigCommerce has a dedicated Slack channel for Technology and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with members of the BigCommerce Product team. To join the BigCommerce developer Slack community, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform). 
```

Comment:
BigCommerce has a dedicated Slack channel for Technology and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with the BigCommerce Product team members. To join the BigCommerce developer Slack community, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform).